packages/@react-spectrum/tabs/src/Tabs.tsx

```diff
@@ -425,7 +423,7 @@ function TabPicker<T>(props: TabPickerProps<T>) {
           items={items}
           ref={ref}
           isQuiet
-          isDisabled={isDisabled}
+          isDisabled={!visible || isDisabled}
```

Comment:
I don't think this is needed but the "should focus the selected tab when tabbing in for the first time" fails arguing that the focus is at the picker who is in a (visually) hidden container

Reply:
I also don't think this is needed. I think the failures are telling us that the accessibility is wrong somewhere.
I'm not quite sure where.
That or RTL/Jest DOM has a bug.

Reply:
So I think this is a JEST-DOM bug. 

Adding the following test cases to the picker tests:
```js
  it('test visibility=hidden', function () {
    render(
      <Provider theme={theme}>
        <div style={{visibility: 'hidden'}}>
          <Picker label="Test" data-testid="test" onSelectionChange={onSelectionChange}>
            <Item>One</Item>
            <Item>Two</Item>
            <Item>Three</Item>
          </Picker>
        </div>
      </Provider>
    );

    userEvent.tab();

    expect(document.activeElement.tagName).toBe('BODY');
  });

  it('test display=none', function () {
    render(
      <Provider theme={theme}>
        <div style={{display: 'none'}}>
          <Picker label="Test" data-testid="test" onSelectionChange={onSelectionChange}>
            <Item>One</Item>
            <Item>Two</Item>
            <Item>Three</Item>
          </Picker>
        </div>
      </Provider>
    );

    userEvent.tab();

    expect(document.activeElement.tagName).toBe('BODY');
  });

  it('test isDisabled', function () {
    render(
      <Provider theme={theme}>
        <div>
          <Picker isDisabled label="Test" data-testid="test" onSelectionChange={onSelectionChange}>
            <Item>One</Item>
            <Item>Two</Item>
            <Item>Three</Item>
          </Picker>
        </div>
      </Provider>
    );

    userEvent.tab();

    expect(document.activeElement.tagName).toBe('BODY');
  });
```
I get the following results:
- test visibility=hidden => fail
- test display=none => fail
- test isDisabled => success

Reply:
@snowystinger so what do we do on this? Keep the work-around with disabling the control or trying to find a work around for the tests.

Reply:
Looks like we might be able to update userEvent https://openbase.com/js/@testing-library/user-event/versions search page for "tab: skip elements with visibility:hidden"
if that's minimal work to update, then we can do here, otherwise, maybe we apply skip with a todo and we update userEvent in a separate PR

Reply:
I tried updateing to user-even v14.2.0 and yes this makes the disabled work-around not needed anymore but there are then tests failing all over the place. So unless I did something really wrong I think updating user-event should be done in a follow up PR.

Reply:
So I think from my side we are done

Reply:
thanks, we're actually on break right now, but we'll be back next week

Reply:
Thanks for the reminder, I'll review it today